src/pages/Invoice/InvoiceModal.js

```diff
@@ -139,9 +141,9 @@ const ChangeCustomerModal = ({
                                                 <p>Item: {info.name}</p>
                                                 <p>Price: ₹{info.price}</p>
                                             </div>
-                                            <button className="btn">
+                                            <Button>
                                                 <FormattedMessage id="select"></FormattedMessage>
-                                            </button>
+                                            </Button>
```

Comment:
Please use <FormattedMessage /> kind of str in the code to make it uniform throughout